src/constructorio-id.js

```diff
@@ -61,6 +63,12 @@
       if (this.cookie_domain) {
         cookie_data += '; domain=' + this.cookie_domain;
       }
+      if (this.cookie_secure) {
+        cookie_data += '; secure';
```

Comment:
This won't affect functionality but it seems it's a convention to write it as capitalized as `Secure`